test.js

```diff
@@ -93,6 +93,7 @@ const types = [
 ];
 
 const names = {
+	'3gp': ['fixture', 'fixture2'],
```

Comment:
should be consistent and not use quotes.

Reply:
@karlhiramoto I'm open to suggestions. However, unquoting `'3gp'` would be a syntax error.

Reply:
Quoted is fine here as it's needed.